src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -488,8 +488,21 @@ def subset_results_by_state(processed_results_dir: str, state: str) -> None:
     abbrev_name_dict = {v: k for k, v in metadata.US_STATE_ABBRV_MAP.items()}
     state_name = sanitize_location(abbrev_name_dict[state.upper()])
     processed_results_dir = Path(processed_results_dir)
-    usa_results_dir = processed_results_dir / "usa"
-    state_dir = processed_results_dir / "states" / state_name
+    dirs = [x for x in processed_results_dir.iterdir() if x.is_dir()]
```

Comment:
Nit: don't abbreviate unnecessarily. Call these variables `directories` and `d` should become `directory`